ceph/ceph_admin/bootstrap.py

```diff
@@ -353,6 +356,7 @@ def bootstrap(self: CephAdmProtocol, config: Dict):
                     cmd = "cephadm shell --"
                     cmd += f" ceph config set mgr mgr/cephadm/container_image_{image}"
                     cmd += f" {override_dict[image_key]}"
+                    logger.info("Executing command : %s", cmd)
```

Comment:
The go forward recommendation is to use `f-string`... could you change it to `logger.info(f"Executing command: {cmd}")`

Reply:
Done.